Makefile

```diff
@@ -10,6 +10,10 @@ build:
 	@echo "++ Building storageos binary"
 	cd cmd/storageos && CGO_ENABLED=0 go build -ldflags "$(LDFLAGS)"
 
+install:
+	@echo "++ Installing storageos binary into \$GOPATH/bin"
+	touch version/client.go && go install -ldflags "$(LDFLAGS)" github.com/storageos/go-cli/cmd/storageos
```

Comment:
`go install -a` will also force re-build, as it supports the same flags as `go build`

Reply:
yes, but apparently a lot slower: https://github.com/golang/go/issues/18369#issuecomment-328705023

I'm fine with either.

Reply:
I'll leave this to your discretion.

Otherwise this PR LGTM